src/context/context.tsx

```diff
@@ -2,26 +2,49 @@ import React, { createContext, ReactNode, useEffect, useReducer } from 'react';
 import { usePosition } from 'hooks/usePosition';
 import { reducer } from './reducer';
 import { CHANGE_POSITION } from './actions';
+import { State, ActionData } from './reducer';
 
-export const initState = {
-  position: {
-    latitude: 0,
-    longitude: 0
+type InitStateType = {
+  state: {
+    position: {
+      latitude: number;
+      longitude: number;
+    };
+    positionError: string;
```

Comment:
1. you can reuse already created `State` interface here
2. i'd recommend not to use common names like State/Action/etc, make them more specific, like PositionContextValues/PositionContextState to avoid duplicating names across the app. also `InitStateType` is not an initial state according to its sense, it's just a type for context values, please rename to see the difference between `State` and `InitStateType` interfaces, because now it looks like both of them are used to describe the same data, but it's not true